misc/lldb_rb/commands/rb_id2str_command.py

```diff
@@ -0,0 +1,49 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class RbID2StrCommand(RbBaseCommand):
+    program = "rb_id2str"
+
+    help_string = "convert and print a Ruby ID to s char * and print it to the lldb console"
```

Comment:
This string is confusing.

```suggestion
    help_string = "convert a Ruby ID to a C string and print it to the LLDB console"
```